armi/reactor/tests/test_components.py

```diff
@@ -494,28 +494,33 @@ def test_changeNumberDensities(self):
 
 
 class TestComponentExpansion(unittest.TestCase):
-    # when comparing to 3D density, the comparison is not quite correct.
-    # We need a bigger delta, this will be investigated/fixed in another PR
+    tCold = 20
     tWarm = 50
     tHot = 500
     coldOuterDiameter = 1.0
 
-    def test_ComponentMassIndependentOfInputTemp(self):
-        coldT = 20
-        circle1 = Circle("circle", "HT9", coldT, self.tHot, self.coldOuterDiameter)
-        coldT += 200
+    def test_ComponentExpansion(self):
```

Comment:
recommend splitting this into multiple tests. this way if all 3 fail we will know vs not knowing if others are passing.

Reply:
Addressed https://github.com/terrapower/armi/pull/942/commits/30cf6d5cbddbe452a043855323f58886491a8fdd